main.js

```diff
@@ -1 +1,381 @@
+app.commandLine.appendSwitch('disable-gpu-vsync')
+app.commandLine.appendSwitch('enable-pointer-lock-options');
+app.commandLine.appendSwitch('disable-accelerated-video-decode', false);
+app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required')
+app.commandLine.appendSwitch('enable-quic');
+app.commandLine.appendSwitch('high-dpi-support', 1);
+
+function init() {
+    //createInitWindow('./loading.html', true, 1, true);
+    createInitWindow('./index.html', true, 1.2, true);
+}
+
+
+//Create Start Window
+function createInitWindow(url, isFullScreen, Size, isMain) {
+    const {
+        width, height
+    } = screen.getPrimaryDisplay().workAreaSize;
+    let initWin = new BrowserWindow({
+        width: width * Size,
+        height: height * Size,
+        show: false,
+
+    });
+
+    initWin.setFullScreen(isFullScreen);
+   
+    initWin.once('ready-to-show', (event) => {
+        initWin.setTitle('NeXi-Client V1.2.9');
+        event.preventDefault();
+        initWin.show();
+    });
+    if (url.indexOf('https://') !== -1) {
+        initWin.loadURL(url);
+    } else {
+        initWin.loadFile(url);
+    }
+    initWin.removeMenu()
+    CheckGame(initWin);
+   
+    if (config.get('utilities_RPC') && isMain){
+        DiscordRPC();
+    }
+    //Just calls Discord RP (WIP).
+    function DiscordRPC() {
+        const rpc = new discord.Client({
+            transport: 'ipc'
+        })
+        rpc.isConnected = false;
+        rpc.login({
+            'clientId': '750116161890287657'
+        })
+        rpc.isConnected = true;
+        rpc.on("ready", () => {
+            rpc.setActivity({
+                largeImageKey: 'nexi-old',
+                details: `${rpc.user.username}`,
+                state: `Started Playing`,
+            })
+            
+            setInterval(() => {
+                updateDiscord();
+            }, 1000);
+            
+        });
+        function updateDiscord(){
+            url = initWin.webContents.getURL();
+            let activity = 'In Menu';
+            if (isMain){
+                if (url.indexOf('index.html#')!== -1){
+                    let arr1 = url.split('/');
+                    let inviteCode = arr1[arr1.length - 1];
+                    inviteCode = inviteCode.replace('index.html#','');
+                    let currentUrl = 'https://venge.io/#';
+                    let arr2 = currentUrl.split(']');
+                    arr2[arr2.length - 1] = `https://venge.io/#${inviteCode}`;
+                    newUrl = arr2.join(']');
+                    
+                    url = newUrl;  
+                    activity = 'Playing Venge.io!';
+                    
+                }
+                else {
+                    if (url.indexOf('index.html') !== -1){
+                        url = rpc.user.username;
+                    }
+                }
+            }
+
+            rpc.setActivity({
+                largeImageKey: 'nexi-old',
```

Comment:
largeImageKey should be changed from nexi-old to nexi-client

Reply:
done